src/effects/TruncSilence.cpp

```diff
@@ -312,9 +313,8 @@ bool EffectTruncSilence::ProcessAll()
    // This list should always be kept in order.
    RegionList silences;
 
-   if (FindSilences(silences,
-      inputTracks()->Selected<const WaveTrack>())
-   ) {
+   if (FindSilences(silences, outputs.Get().Any<const WaveTrack>()))
```

Comment:
What if output tracks contains un-selected tracks of the sync lock group?  They would be analyzed too

Reply:
I checked for the other use of FindSilences but that is correctly left unchanged as it already applies to an output track.